src/services/notifier/utils/updaterUtils.ts

```diff
@@ -52,17 +55,20 @@ export const updateSubscriptionsBy = async (
 ): Promise<void> => {
   const [host, port] = providerUrl.split(/(?::)(\d*)$/, 2)
   const svcProvider = new NotifierSvcProvider({ host, port })
-  const subscriptionsDTO: any[] = await svcProvider.getSubscriptions(consumerAddress, subsHashesToUpdate)
-
-  await Promise.all(subscriptionsDTO.map(({
-    hash,
-    status,
-    paid,
-    notificationBalance
-  }) =>
-    SubscriptionModel.update(
-      { status, paid, notificationBalance },
-      { where: { hash } }
-    )
-  ))
+  try {
+    const subscriptionsDTO: any[] = await svcProvider.getSubscriptions(consumerAddress, subsHashesToUpdate)
+    await Promise.all(subscriptionsDTO.map(({
+      hash,
+      status,
+      paid,
+      notificationBalance
+    }) =>
+      SubscriptionModel.update(
+        { status, paid, notificationBalance },
+        { where: { hash } }
+      ).catch(error => logger.warn(`Unable to update subscription with hash ${hash} in the database`, error))
+    ))
+  } catch (error) {
+    logger.warn('Unable to update subscriptions from notifier service provider', error)
+  }
```

Comment:
How about this @farinalvaro?

```suggestion
  const logError = (error) => logger.warn('Unable to update subscriptions from notifier service provider', error)
  
  const subscriptionsDTO: any[] = await svcProvider
  .getSubscriptions(consumerAddress, subsHashesToUpdate)
  .catch(logError)
  
  await Promise.all(subscriptionsDTO.map(({
    hash,
    status,
    paid,
    notificationBalance
  }) =>
    SubscriptionModel.update(
      { status, paid, notificationBalance },
      { where: { hash } }
    ).catch(logError)
  ))
 
```

Reply:
The problem with this one is that if the first one fails it keeps going and `subscriptionsDTO` will be `undefined`, and then it will crash “`subscriptionsDTO.map` cannot read property map of undefined”.